siliconcompiler/core.py

```diff
@@ -2779,7 +2787,9 @@ def check_logfile(self, jobname=None, step=None, index='0',
                             else:
                                 self.logger.info(f'{suffix}: {line_with_num}')
 
-        for suffix in checks:
+        for suffix in ordered_suffixes:
+            if display:
+                self.logger.info(f'Number of {suffix}: {matches[suffix]}')
```

Comment:
I think we still want the summary, just not the details

Reply:
now printing regardless of `display`